crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -43,154 +42,464 @@ mod validate_op_tests;
 
 #[instrument(skip(
     workspace,
-    space,
+    incoming_dht_ops_sender,
+    current_validation_dependencies,
     trigger_app_validation,
-    sys_validation_trigger,
+    trigger_self,
     network
 ))]
-pub async fn sys_validation_workflow<Network: HolochainP2pDnaT + Clone + 'static>(
+pub async fn sys_validation_workflow<
+    Network: HolochainP2pDnaT + Clone + 'static,
+    Sender: DhtOpSender + Send + Sync + Clone + 'static,
+>(
     workspace: Arc<SysValidationWorkspace>,
-    space: Arc<Space>,
+    incoming_dht_ops_sender: Sender,
+    current_validation_dependencies: Arc<Mutex<ValidationDependencies>>,
     trigger_app_validation: TriggerSender,
-    sys_validation_trigger: TriggerSender,
+    trigger_self: TriggerSender,
     network: Network,
 ) -> WorkflowResult<WorkComplete> {
-    let complete =
-        sys_validation_workflow_inner(workspace, space, network, sys_validation_trigger).await?;
+    // Run the actual sys validation using data we have locally
```

Comment:
There are significant workflow changes at this level, I've commented to explain what it's doing but this section is probably the most significant to review